README.md

```diff
@@ -1,70 +1,87 @@
-      java_opts:
-        - "-XX:MaxPermSize=128m"
-        - "-Xmx256m"
+Generally, you will only use a jetty instance for your project's web service dependencies during development and testing, not for production. So you would add this to your Gemfile:
 
-You can also configure a specific version of hydra-jetty. This is placed in your codebase, usually in a rake task 
+```
+group :development, :test do
+  gem 'jettywrapper'
+end
+```
 
-    Jettywrapper.hydra_jetty_version = "v1.2.3"
+Or, if your project is a gem, you would add this to your .gemspec file:
+
+```
+Gem::Specification.new do |s|
+  s.add_development_dependency 'jettywrapper'
+end
+```
 
-Alternatively, you can use a completely different Jetty-based repository, Hydra-related or not
+Then execute:
 
-    Jettywrapper.url = "https://github.com/myorg/my-jetty/archive/master.zip"
+    $ bundle
 
-The name of the zip file can either be a branch name, such as master or develop, or the tag name of a released version.
-Basically, any url that Github provides as a *.zip file will work.
+Or install it yourself as:
 
-## Example rake task
+    $ gem install jettywrapper
 
-```ruby
-require 'jettywrapper'
-Jettywrapper.url = "https://github.com/myorg/my-jetty/archive/testing-feature-branch.zip"
-desc "Hudson build"
-task :hudson do
-  jetty_params = Jettywrapper.load_config.merge({:jetty_home => File.expand_path(File.dirname(__FILE__) + '/../jetty')})
-  error = Jettywrapper.wrap(jetty_params) do  
-    Rake::Task["spec"].invoke
-  end
-  raise "test failures: #{error}" if error
-end
-```
 
-## Testing the gem 
+## Usage
 
-If you haven't already, clone the git repository
+### Configuration
 
-    git clone git@github.com:projecthydra/jettywrapper.git
-    cd jettywrapper
+See [Configuring jettywrapper](https://github.com/projecthydra/jettywrapper/wiki/Configuring-jettywrapper).
 
-Install the gems
+If you don't need both Solr and Fedora, we recommend you avoid the default [hydra-jetty](https://github.com/projecthydra/hydra-jetty). If you only need Solr, use [blacklight-jetty](https://github.com/projectblacklight/blacklight-jetty).
 
-    bundle install
+### Gotchas
+
+* Jetty may take a while to spin up
+* Jetty may not shut down cleanly
+
+See [Using jettywrapper](https://github.com/projecthydra/jettywrapper/wiki/Using-jettywrapper) for more information and what to do.
+
+### Example Rake Task
+
+See [Using jettywrapper](https://github.com/projecthydra/jettywrapper/wiki/Using-jettywrapper) for more information.
+
+```ruby
+require 'jettywrapper'
+
+desc 'run the tests for continuous integration'
+task ci: ['jetty:clean', 'myproj:configure_jetty'] do
+  ENV['environment'] = 'test'
+  jetty_params = Jettywrapper.load_config
+  jetty_params[:startup_wait] = 60
+
+  error = nil
+  error = Jettywrapper.wrap(jetty_params) do
+    # run the tests
+    Rake::Task['spec'].invoke
+  end
+ raise "test failures: #{error}" if error
+end
+```
 
-Run the tests
+## Contributing
 
-    rake 
+1. Fork it
```

Comment:
Instead of this, how about pulling in this file and linking to it here: https://github.com/projecthydra/hydra/blob/master/CONTRIBUTING.md